docs/pages/includes/database-access/reference/rds-action-notes-with-modify.mdx

```diff
@@ -0,0 +1,9 @@
+{{}}
```

Comment:
Is this a bug workaround? Do we have a GitHub issue for it?

Reply:
yes. I have removed the hack and created an issue https://github.com/gravitational/docs/issues/461